.github/workflows/deploy-template.yml

```diff
@@ -87,8 +87,8 @@ jobs:
           envs=( ${{ needs.prepare-values.outputs.environments }} )
           for env in ${envs[*]};
           do
-            yq e -i '.spec.template.spec.containers.[0].image = "008577686731.dkr.ecr.us-gov-west-1.amazonaws.com/dsva/${{inputs.ecr_repository}}:${{inputs.ecr_repository}}-${{ github.sha }}"' $env/deployment.yml
-            yq e -i '.spec.template.spec.initContainers.[0].image = "008577686731.dkr.ecr.us-gov-west-1.amazonaws.com/dsva/${{inputs.ecr_repository}}:${{inputs.ecr_repository}}-${{ github.sha }}"' $env/deployment.yml
+            yq e -i '(.spec.template.spec.containers[] | select(.name == "platform-console-api").image) |= "008577686731.dkr.ecr.us-gov-west-1.amazonaws.com/dsva/${{inputs.ecr_repository}}:${{inputs.ecr_repository}}-${{ github.sha }}"' $env/deployment.yml
+            yq e -i '(.spec.template.spec.initContainers[] | select(.name == "platform-console-api").image) |= "008577686731.dkr.ecr.us-gov-west-1.amazonaws.com/dsva/${{inputs.ecr_repository}}:${{inputs.ecr_repository}}-${{ github.sha }}"' $env/deployment.yml
```

Comment:
Should these be variables instead? Just wondering if we should hardcode `platform-console-api` if we are going to be using this template for other repos.

Reply:
Oh good idea. I like the variable suggestion

Reply:
@rileyanderson Fixed ✔️